ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -604,3 +604,72 @@ literals:
 
 # Added 4/12/20
 - supply_air_damper_percentage_sensor
+
+# Added 9/8/20 for SG-SIN-MBC2
+- chilled_return_water_flowrate_sensor
+- device_fault_status
+- fire_alarm_status
+- fire_fault_status
+- makeup_water_high_tank_percentage_status
+- makeup_water_low_tank_percentage_status
+- max_cooling_air_flowrate_setpoint
+- min_cooling_air_flowrate_setpoint
+- pump_fault_status
+- pump_run_status
+- recovery_alarm_command
+- schedule_command
+- photocatalyst_filter_run_status
+- water_high_tank_alarm_status
+- water_leak_alarm_status
+- water_leak_detection_fault_status
+- zone_command
+# - automatic_mode
+# - ballast_status
+
+# Added 9/14/20 for TW-NTC-TPKD
+- condensing_water_valve_percentage_sensor
+- primary_speed_percentage_command
+- primary_speed_percentage_sensor
+- secondary_speed_percentage_command
```

Comment:
are these primary and secondary speeds associated with pumps? would you not model those as individual components? seems like `speed_percentage_command` is sufficient in that model

Reply:
Yes it is associated with pump. I agree with you to use only "speed_percentage_command" & "Speed_percentage_sensor" for both.